Components/NavigationMain/_style.scss

```diff
@@ -26,8 +26,7 @@
     height: 100%;
 
     &-image {
-      height: 100%;
-      width: auto;
+      max-width: 130px;
```

Comment:
The logo used `height: 100%` before to not change / break the height of the navigation.
What's your intention to change it to `max-width` instead?

Reply:
You are right, that was mainly for IE bug but i found a more generic way that works with all images i tried